### DISCUSSION TITLE
Pet Profiles Team Staff (5/5)

### DISCUSSION THREAD

**Discussion by rukitoyordan** [uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp#6] [Pet Profiles Team]
### Pet Profiles Team for Milestone No.1 
**Main Manager**
- Evand Sanchez @evnd04 

**Team Leader**
- Alejandro J. De Jesus (@Alejandro-DeJesus)

**Developers**
- Alexis Traverso (@AlexisTraversoPR)
- Alejandro J. De Jesus (@Alejandro-DeJesus)
- Raquel Quiñones (@RaquelQuinones)
- Rayane Mili (@rayanemili)
- Alejandro Diaz (@AleDiazz)

**Comment by RaquelQuinones** [comment]
Hello! I was wondering when everyone is free to meet up and talk about the task for the Pet Profiles. Pueden durante la hora universal el jueves?

**Reply by rukitoyordan** [reply]
@RaquelQuinones do you wish to be the Team Leader? If so let me know or if anyone volunteers! What I need is for y'all to meet and set tasks for your team in the "Issues" before next Friday's class. Preferably, **Thursday February 8th, 2024 @ 11:59PM.**

**Reply by Alejandro-DeJesus** [reply]
Yo puedo a esa hora

**Reply by RaquelQuinones** [reply]
@rukitoyordan atm I can't be team leader, but thank you for considering me.

**Reply by rukitoyordan** [reply]
That's alright @RaquelQuinones.
@Alejandro-DeJesus @rayanemili @AleDiazz, alguno esta interesado en ser Team Leader? Entiendo que no muchos tengan luz, so seremos mas flexibles con los deadlines, pero deben ir añadiendo tasks y organizarze lo antes posible!

**Reply by Alejandro-DeJesus** [reply]
@RaquelQuinones Como hacemos la reunión del jueves, de manera presencial (en ese caso donde) o manera online?

**Reply by RaquelQuinones** [reply]
@Alejandro-DeJesus Manera Online; voy a mandar un link por el chat de whatapp.

**Reply by rukitoyordan** [reply]
Recuérdenme por favor antes de reunirse quien será Team Leader de su equipo!

**Reply by rukitoyordan** [reply]
Reminder you have until TODAY to decide who will be the Team Leader. Otherwise, I will pick a random developer from the team. ‼️‼️

**Comment by Alejandro-DeJesus** [comment]
[Minuta INSO 4101.docx](https://github.com/uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp/files/14303859/Minuta.INSO.4101.docx)

**Comment by RaquelQuinones** [comment]
[MInuta PetProfile INSO Meeting #1.pdf](https://github.com/uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp/files/14364986/MInuta.PetProfile.INSO.Meeting.1.pdf)

**Comment by Alejandro-DeJesus** [comment]
[Minuta 2-20-2024.pdf](https://github.com/uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp/files/14390903/Minuta.2-20-2024.pdf)
